src/components/button/button.scss

```diff
@@ -1,5 +1,8 @@
-@import '../../style/variables';
 @import "@limetech/mdc-button/mdc-button";
+@import '../../style/mdc-imports';
+@import '../../style/mdc-typography';
```

Comment:
The order of these imports are important, because the variables have to be imported before the component styles to have any effect. So the the button doesn't quite look like we want it to with this change 😄

(I'll fix that of course, since I'm working on the PR. But I wanted to let you know for the future 👍)

Reply:
![image](https://user-images.githubusercontent.com/154725/76407472-b070ae00-638b-11ea-94bb-0d3940d87ba7.png)
![image](https://user-images.githubusercontent.com/154725/76407445-a51d8280-638b-11ea-8245-3f4cf2246492.png)
![image](https://user-images.githubusercontent.com/154725/76407454-a9e23680-638b-11ea-865e-05efa28211c7.png)

Reply:
Ah, the last two screenshots was actually because I had my watcher running while fiddling with the files. But the all-caps on the first screenshot is there in the PR.

Or that change was intentional?